string emotion
string[] w_emotions
float64[] probs
